### PR TITLE
Include Cosmonaut Stakes Relayer in docs

### PR DESCRIPTION
**Pull request by napelvs**
- Include Cosmonaut Stakes as a Relayer between Persistence and:
      - Gravity Bridge

- Include Cosmonaut Stakes Relayer Contact Information.

### CODE REVIEW

File: build/relayers/ibc-relayers.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ We encourage everyone to follow and support our IBC-relayer operators
...
+| [Cosmonaut Stakes](https://twitter.com/CosmonautStakes)   | <p><img src="https://user-images.githubusercontent.com/95667791/163506878-e7e278b2-1cf6-4dac-bc13-e92f63600ee0.png" alt="Twitter" data-size="line"> **Twitter:** <a href="https://twitter.com/CosmonautStakes">@CosmonautStakes</a><br><img src="https://user-images.githubusercontent.com/95667791/163505650-fe08f2a7-8419-4a2f-97b7-e5182dd386e1.png" alt="Discord" data-size="line"> **Discord:** Danoly \| Cosmonaut Stakes#6003<br><img src="https://user-images.githubusercontent.com/95667791/163506639-7d27e947-94c7-4740-bb46-8e5791e9fc6c.png" alt="Telegram" data-size="line"> **Telegram:** <a href="https://t.me/CosmonautStakes">@CosmonautStakes</a></p>                             | <p>🔒 Delegate to **Cosmonaut Stakes** on: <a href="https://wallet.keplr.app/chains/persistence?modal=validator&chain=core-1&validator_address=persistencevaloper1rzauu3undh97yvdnj7wu2wwstm9wj8heeq2vcz&referral=true">Persistence</a></p>                                                                                                                                                                                                               |
```
Comment (ajeet97) [inline build/relayers/ibc-relayers.md:18]: The image is very big in preview. Can add `width` b/w 20-30 in the `img` tag (whichever looks fine).
Also if you can add width for all the rows, it'd be great.

Comment (napelvs) [reply build/relayers/ibc-relayers.md]: I've updated the width of all images and added an emoji for the email to follow the same pattern as the other social media links. I hope this is what you were referring to when saying "**add width for all the rows**".

Comment (ajeet97) [reply build/relayers/ibc-relayers.md]: Great, Thank 👍

**Review by ajeet97 — APPROVED** [review]
LGTM

**Pull request merged**